src/styles/blocks/day.scss

```diff
@@ -0,0 +1,26 @@
+@import "../utils/variables";
+
+.day {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  width: $cell-size;
+  height: $cell-size;
+  background-color: #eee;
+  box-sizing: border-box;
+  border: 1px solid black;
+  transition: 0.3s;
```

Comment:
It's better to specify to what properties you need to apply the transition.
![image](https://github.com/mate-academy/layout_calendar/assets/63243136/ca3d0823-db54-4fb2-8bda-90df82574b0d)